examples/README.md

```diff
@@ -46,6 +46,12 @@ _Add yours here!_
 * **Description:** A task manager that lets you create background tasks asynchronously, and check their status at any point in time.
```

Comment:
Let's move everything to:

https://github.com/winglang/examples

@skorfmann created an awesome framework there for maintaining these

Reply:
yes, seems like a great addition for the examples over there :)